apps/server/src/modules/tool/service/context-external-tool.service.ts

```diff
@@ -89,4 +90,13 @@ export class ContextExternalToolService implements AuthorizationLoaderService {
 	findById(id: EntityId): Promise<ContextExternalToolDO> {
 		return this.getContextExternalToolById(id);
 	}
+
+	async getContextExternalToolsForContext(contextType: ToolContextType, contextId: string) {
```

Comment:
Could that also be `findAllByContext` or `getAllForContext` etc. It looks like the method's purpose is to obtain a list of `ContextExternalTool`s for a spcific context.

Reply:
Good idea :)